components/back-link/src/__snapshots__/test.js.snap

```diff
@@ -41,27 +42,21 @@ exports[`Back Link matches wrapper snapshot: wrapper mount 1`] = `
   bottom: 1px;
   left: 0;
   margin: auto;
-  -webkit-clip-path: polygon(0% 50%, 100% 100%, 100% 0%);
 }
 
-.glamor-0:focus,
-[data-glamor-0]:focus {
+.emotion-1:focus {
   background-color: #ffbf47;
   outline: 3px solid #ffbf47;
 }
 
-@media only screen and (min-width: 641px) {
-  .glamor-1,
-  [data-glamor-1] {
-    margin-bottom: 15px;
-  }
+.glamor-0,
```

Comment:
why is there still a glamor class in the snapshot?

Reply:
It's part of the HoC for spacing `withWhiteSpace`. I initially thought the work would just be swapping it out on the components but I realised half way through there were other areas it will need to be swapped out. I'll go through this today as I feel confident this change will not introduce any scary / strange bugs.